elastic/integrations
# Parse out system Integration, create Linux integration

## What does this PR do?

This is in preparation for 7.10. We want to strip the system integration down to just cross-compatible datasets. This will be the first of a few changes for 7.10, the rest will involve cleaning up the existing cross-compatible datasets.

## Checklist


## Author's Checklist

- [ ] Do we want a custom icon for the Linux Integration?

## How to test this PR locally

- Pull down the PR
- Build the new package registry
- Enroll a test agent, make sure the `linux` and `system` integrations are properly ingesting data
